index.js

```diff
@@ -101,8 +110,8 @@ function main(argv, callback) {
   });
 
   app.on('listening', function(listenAddr) {
-    console.log('%s: listen on %s, work base is `%s`',
-      $0, listenAddr.port, base);
+    console.log('%s: listen on %s, work base is `%s`, driver is `%s`',
+      $0, listenAddr.port, base, app.getDriverInfo().type);
```

Comment:
The docker version does. That's what the filtering is for, emitting a `listening` event for the first `request {cmd: 'listening'}` from an instance.